packages/calypso-sentry/src/index.ts

```diff
@@ -218,4 +248,16 @@ export async function initSentry( { beforeSend, userId }: SentryOptions ) {
 		// Either way they don't serve a purpose.
 		clearQueues();
 	}
+
+	return 'disabled'; // Previous state: disabled -> We were not running [initial, disabled]
```

Comment:
If we go with the approach I mentioned, we can remove all extra return statements.